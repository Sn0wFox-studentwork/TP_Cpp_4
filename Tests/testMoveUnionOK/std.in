R r1 2 2 1 1
R r2 1 1 0 0
OR r12 r1 r2
MOVE r12 1 1
LIST
EXIT