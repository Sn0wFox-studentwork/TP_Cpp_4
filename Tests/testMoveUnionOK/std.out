OK
OK
OK
OK
R r1 2 2 1 1
OR r12 {
R 3 3 2 2
R 2 2 1 1
}
R r2 1 1 0 0